airbyte-integrations/connectors/source-hardcoded-records/pyproject.toml

```diff
@@ -16,8 +16,8 @@ repository = "https://github.com/airbytehq/airbyte"
 include = "source_hardcoded_records"
 
 [tool.poetry.dependencies]
-python = "^3.10,<4.0"
-airbyte-cdk = "*"
+python = ">=3.10,<4.0"
+airbyte-cdk = "6.6.1"
```

Comment:
```suggestion
airbyte-cdk = ">=6.6.1,<7.0"
```

Reply:
It ended up pinning 6.6.1 for some reason still